packages/manager/apps/cloud/client/app/dbaas/logs/detail/inputs/add/edit/logs-inputs-add-edit.controller.js

```diff
@@ -95,9 +96,13 @@ class LogsInputsAddEditCtrl {
     return this.inputAddEdit
       .load()
       .then((successData) => {
-        this.gotToNextStep(this.inputId || successData[0].item.inputId);
+        if (successData[0].item) {
+          this.gotToNextStep(this.inputId || successData[0].item.inputId);
+        } else {
+          this.$q.reject();
+        }
       })
-      .catch(() => this.CucControllerHelper.scrollPageToTop());
+      .finally(() => this.CucControllerHelper.scrollPageToTop());
```

Comment:
It seems that you could kept the `catch` call, as you either call `$q.reject` or a `$state.go`